docs/manual-EN/3.build-develop-and-administration/3.configurations/8.single-node-log.md

```diff
@@ -0,0 +1,61 @@
+# Logs
+
+**Nebula Graph** uses [glog](https://github.com/google/glog) to print logs, [gflag](https://gflags.github.io/gflags/) to control the severity level of the log, and provides an HTTP interface to dynamically change the log level at runtime to facilitate tracking.
+
+## Log Directory
+
+The default log directory is `/usr/local/nebula/logs/`.
+
+**Note:** If you deleted the log directory during runtime, the runtime log would not continue to be printed. However, this operation will not affect the services. Restart the services to recover the logs.
+
+## Parameter Description
+
+### Two most commonly used flags in glog
+
+- minloglevel 0-3: the numbers of severity levels INFO(DEBUG), WARNING, ERROR, and FATAL are 0, 1, 2, and 3, respectively. Usually specified as 0 in debug, 1 in production. If you set the minloglevel to 4, no logs are printed.
+- v 0-4: when the v is set to 0,  you can further set the severity level of the debug log. The larger the value, the more detailed the log.
+
+### Configuration Files
+
+The default severity level for the metad, graphd, and storaged logs can be found in the configuration files (usually in `/usr/local/nebula/etc/`).
+
+## Check and Change the Severity Levels Dynamically
+
+Check all the flag values (log values included) of the current glags with the following command.
+
+```bash
+> curl ${ws_ip}:${ws_port}/get_flags
+```
+
+In the command:
+
+- `ws_ip` is the HTTP service IP, which can be found in the above configuration files (default IP is 127.0.0.1).
+- `ws_port` is the HTTP port, the default value for `metad` is 11000, `storaged` is 12000， and `graphd` is 13000.
```

Comment:
is to specify the port for the HTTP service. for `storaged` is ..., and for `graphd` is ...